shell/src/main/java/tachyon/shell/TFsShell.java

```diff
@@ -116,45 +112,35 @@ public int cat(TachyonURI path) throws IOException {
    * Load a file or directory in Tachyon space, makes it resident in memory.
    *
    * @param filePath The TachyonURI path to load into Tachyon memory
-   * @return 0 if command is successful, -1 if an error occurred.
    * @throws IOException
    */
-  public int load(TachyonURI filePath) throws IOException {
+  public void load(TachyonURI filePath) throws IOException {
     TachyonFS tachyonClient = createFS(filePath);
-    int ret = loadPath(tachyonClient, filePath);
-    if (ret == 0) {
+    try {
+      loadPath(tachyonClient, filePath);
       System.out.println(filePath + " loaded");
-    } else {
-      System.out.println("Loading " + filePath + " failed");
+    } catch (IOException e) {
+      throw new IOException("Loading " + filePath + " failed");
     }
-    return ret;
   }
 
-  private int loadPath(TachyonFS tachyonClient, TachyonURI filePath) throws IOException {
+  private void loadPath(TachyonFS tachyonClient, TachyonURI filePath) throws IOException {
     TachyonFile tFile = tachyonClient.getFile(filePath);
     if (tFile == null) {
-      return -1;
+      throw new IOException(filePath + " does not exist.");
```

Comment:
To save the boilerplate code of exception messages, you can add an entry to the enum ExceptionMessage.